enhancements/ingress/set-delete-http-headers.md

```diff
@@ -1002,16 +1006,27 @@ This route can be annotated with `haproxy.router.openshift.io/disable_cookies=tr
   IngressController.
 
 #### Why does a custom header value defined in RouteSpec override IngressControllerSpec value ?
-  Answer: It's quite common to have a global setting with an escape hatch locally to override the global setting.
-  The purpose of this EP is to define custom headers. So, there are two reasons we need such override:
-  1. A route owner shall have the choice to define what headers he wants as per his application requirements(as they might be varying) 
-  than the ones admin has set cluster-wide.
-  2. The IngressController and Route might be owned by the same person who wants to specify some global configuration
-  and then override that global configuration on a small subset of routes. 
```

Comment:
I would keep this section in here, not move it to the Alternatives section.

Reply:
i think this comes under upside

Reply:
I don't understand?

Reply:
i think we will need to remove this as we no-longer use this over-ride behavior where route over-rides ingresscontroller in all cases.